php/class-fieldmanager-gallery.php

```diff
@@ -83,7 +83,13 @@ class Fieldmanager_Gallery extends Fieldmanager_Field {
 	 * @param string $label
 	 * @param array  $options
 	 */
-	public function __construct( $label, $options = array() ) {
+	public function __construct( $label = '', $options = array() ) {
+
+		if ( is_array( $label ) ) {
+			$options = $label;
+		} else {
+			$options['label'] = $label;
```

Comment:
This logic IS handled in the parent constructor which is called on line 130, but since it isn't handled here, the button labels won't be correct when a field is created with the options as the first parameter.